tests/e2e/singlecard/models/configs/Qwen3-30B-A3B.yaml

```diff
@@ -15,4 +15,5 @@ gpu_memory_utilization: 0.6
 enable_expert_parallel: True
 tensor_parallel_size: 2
 apply_chat_template: False
-fewshot_as_multiturn: False
\ No newline at end of file
+fewshot_as_multiturn: False
+parallel_mode: "TP2 + EP"
```

Comment:
Please don't add any redundant info in yaml, becuase it will bring some comlex when adding

such as parallel_mode , can be generate from tensor_parallel_size and enable_expert_parallel